play-media-native/screens/EditAthleteDetails.tsx

```diff
@@ -123,6 +130,16 @@ export const EditAthleteDetailsScreen = ({ route, navigation }) => {
             caretHidden
             onTouchStart={() => setShowCareerStartDatePicker(true)}
```

Comment:
The previous one was forgotten here  
![image](https://user-images.githubusercontent.com/96472119/219349841-0e332fd5-5f2f-4841-bec5-fae9cb358de8.png)